### DISCUSSION TITLE
Packages which are there in MS Store should be removed from here

### DISCUSSION THREAD

**Discussion by vedantmgoyal9** [microsoft/winget-pkgs#29985] [Q&A] [answered]
There are some packages which are present in Microsoft Store as well as in this repository.

The question is "Should we remove these packages from here to avoid duplicate conflicts while running `winget install` as winget can now install packages from MS Store directly"?

**Comment by denelon** [comment, marked as answer]
We have been discussing catalog deduplication. We still need to improve matching in the "**msstore**" source, and a couple of other Issues:

https://github.com/microsoft/winget-cli/issues?q=is%3Aissue+is%3Aopen+label%3Amsstore

One of the other challenges is the store only retains a single current version. If these packages are removed from the community repository, the historical versions are no longer available. That may be fine for the vast majority of packages, but some of the developer tools may need to be retained.

**Comment by jedieaston** [comment]
Also, some packages use different installers for their Microsoft Store deployments (even for Win32). VSCode uses some weird bootstrapper so they don't have to update the Microsoft Store for every release, for example. (Of course, Win32 apps should be retained indefinitely since Windows 10 can't install them from the Microsoft Store).

**Comment by vedantmgoyal9** [comment]
I am not talking about Win32 apps since they aren't supported on Windows 10 until a big update comes for Windows 10 which won't happen probably (2/10 probability). Packages like `Automattic.Simplenote` etc.., fit neither in the Win32 apps category nor in "_That may be fine for the vast majority of packages, but some of the developer tools may need to be retained._"

"

**Reply by jedieaston** [reply]
Win32 support is coming to the Microsoft Store for Windows 10. https://www.theverge.com/2021/10/27/22748243/microsoft-store-windows-10-update-windows-11-app-store

It literally uses winget under the covers (you can check your winget logs folder, it puts them in the same place).

**Comment by Trenly** [comment]
I personally don’t think its an issue because we know there will be some form of default [Behavior when Multiple Sources are Configured](https://github.com/microsoft/winget-cli/issues/1488). I personally would rather use the version from the community repo than the one that is in the store, even if they are exactly the same. I’m sure others would rather use the one from the store. 

I believe having it available in both sources would be more beneficial, since it would give more choice to the users

**Comment by vedantmgoyal9** [comment]
I didn't saw that. If a user can set priority on sources, then it is probably fine.
